domino_generator/lib/src/component_generator.dart

```diff
@@ -94,7 +94,12 @@ class ComponentGenerator {
       defaultInits.forEach(_sb.writeln);
       topLevelObjects.add('\$d');
 
-      _render(Stack(objects: topLevelObjects), template.nodes);
+      // SCSS class name
```

Comment:
please add a TODO to: consider hashing the style content and appending that to the class name.